### ISSUE TITLE
Support 'repeatcount' in Python post-processing scripts

### ISSUE THREAD

**Issue by BrechtDeMan** [BrechtDeMan/WebAudioEvaluationTool#268]
Currently pages which are repeated by using the `repeatcount` attribute lead to an error when running such scripts. 

This should add multiple lines from the same subject to the per-page result CSV, e.g. abc123-0, abc123-1, ...

**Comment by nickjillings** [comment]
Yep, this needs addressing, along with a number of other bugs in the python parsers.

**Comment by BrechtDeMan** [comment]
I need to do a rewrite. Will do it as soon as I can allocate some time to this.
